### ISSUE TITLE
UnicodeDecodeError with eb logs

### ISSUE THREAD

**Issue by paklau** [aws/aws-elastic-beanstalk-cli#37]
### Description

When using `eb logs`, UnicodeDecodeError is thrown and no logs can be viewed

### Steps to reproduce

`eb logs`

### Observed result
The api request returned 200. But it seems the eb cli cannot display the returned logs.

```
2020-07-08 00:44:53,898 (INFO) eb : Traceback (most recent call last):
  File "/usr/local/Cellar/aws-elasticbeanstalk/3.18.1/libexec/lib/python3.8/site-packages/ebcli/core/ebrun.py", line 62, in run_app
    app.run()
  File "/usr/local/Cellar/aws-elasticbeanstalk/3.18.1/libexec/lib/python3.8/site-packages/cement/core/foundation.py", line 797, in run
    return_val = self.controller._dispatch()
  File "/usr/local/Cellar/aws-elasticbeanstalk/3.18.1/libexec/lib/python3.8/site-packages/cement/core/controller.py", line 472, in _dispatch
    return func()
  File "/usr/local/Cellar/aws-elasticbeanstalk/3.18.1/libexec/lib/python3.8/site-packages/cement/core/controller.py", line 478, in _dispatch
    return func()
  File "/usr/local/Cellar/aws-elasticbeanstalk/3.18.1/libexec/lib/python3.8/site-packages/ebcli/core/abstractcontroller.py", line 92, in default
    self.do_command()
  File "/usr/local/Cellar/aws-elasticbeanstalk/3.18.1/libexec/lib/python3.8/site-packages/ebcli/controllers/logs.py", line 65, in do_command
    self.__get_logs()
  File "/usr/local/Cellar/aws-elasticbeanstalk/3.18.1/libexec/lib/python3.8/site-packages/ebcli/controllers/logs.py", line 99, in __get_logs
    logsops.retrieve_beanstalk_logs(
  File "/usr/local/Cellar/aws-elasticbeanstalk/3.18.1/libexec/lib/python3.8/site-packages/ebcli/operations/logsops.py", line 486, in retrieve_beanstalk_logs
    get_logs(env_name, info_type, do_zip=do_zip, instance_id=instance_id)
  File "/usr/local/Cellar/aws-elasticbeanstalk/3.18.1/libexec/lib/python3.8/site-packages/ebcli/operations/logsops.py", line 345, in get_logs
    _handle_tail_logs(instance_id_list)
  File "/usr/local/Cellar/aws-elasticbeanstalk/3.18.1/libexec/lib/python3.8/site-packages/ebcli/operations/logsops.py", line 889, in _handle_tail_logs
    data.append(utils.decode_bytes(log_result))
  File "/usr/local/Cellar/aws-elasticbeanstalk/3.18.1/libexec/lib/python3.8/site-packages/ebcli/lib/utils.py", line 378, in decode_bytes
    value = value.decode('utf8')
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xfc in position 9577: invalid start byte

2020-07-08 00:44:53,898 (INFO) eb : UnicodeDecodeError - 'utf-8' codec can't decode byte 0xfc in position 9577: invalid start byte
```
### Expected result

Logs

### Additional environment details (Ex: Windows, Mac, Amazon Linux etc)

MacOS 10.15.5
iTerm2
EB CLI 3.18.1 (Python 3.8.3)

**Comment by alexandargyurov** [comment]
Looks very similar to an issue we had: #15

**Comment by Palakpatel67** [comment]
Hi @paklau 
Thanks for reporting this issue. We will look into this and post any updates to this issue.

**Comment by lwiddershoven** [comment]
I have the same issue, on linux. 
As this is my very first experience with eb (and aws for that matter) and my app doesn't do what I expected it really is a nuisance.

```
leon@xyz:~/Development/spring-boot-aws-1$ eb logs
Retrieving logs...
ERROR: UnicodeDecodeError - 'utf-8' codec can't decode byte 0xe0 in position 20145: invalid continuation byte
leon@xyz:~/Development/spring-boot-aws-1$ eb logs
Retrieving logs...
ERROR: UnicodeDecodeError - 'utf-8' codec can't decode byte 0xe0 in position 1698: invalid continuation byte
leon@xyz:~/Development/spring-boot-aws-1$ eb logs
Retrieving logs...
ERROR: UnicodeDecodeError - 'utf-8' codec can't decode byte 0xe0 in position 1698: invalid continuation byte
```
I'm using Python 3.8.5, eb was compiled against a python 3.7.2 that was downloaded by the install scripts. No clue if that matters or not but it may be a library thing.

**Comment by OgnjenAnari** [comment]
Exect same issue as @lwiddershoven. Thanks for `eb logs -a` btw, that indeed works.

**Comment by Palakpatel67** [comment]
This ishould be fixed in awsebcli version [3.19.3](https://pypi.org/project/awsebcli/3.19.3/).
